### DISCUSSION TITLE
SMB share

### DISCUSSION THREAD

**Discussion by mash2k3** [itsToggle/plex_debrid#306] [General]
Is it possible to access the realdebrid volume created via rclone_rd using smb share?

I would like to test plex_debrid with Kodi

**Comment by itsToggle** [comment]
It would be great if you'd join the discord server for small questions like this, so that the discussion tab doesn't get flooded. Check out rclones "serve" command. You can serve a DNLA or a WebDAV server as far as I know.

**Reply by mash2k3** [reply]
thanks, ill check discord

**Comment by mash2k3** [comment]
If anybody is curious, i was able to mount the realdebrid volume using command below
`rclone mount /var/lib/docker/plugins/'volume-ID'/propagated-mount/realdebrid/ /mnt/user/appdata/Debrid --dir-cache-time 10s --allow-other --allow-non-empty`

Might not be the right way, i am open to other better methods.

**Reply by itsToggle** [reply]
Its generally not a good idea to use allow-non-empty, as this can cause problems if there is a mount to that directory already. This can occur if an instance of rclone was not closed properly, so it's best to remove that tag.
 
I'm not sure why you would specify a docker volume location - did you want to mount the docker volume plugin to your host OS? If you already have the binary version of rclone (which you seem to have since you can call the "rclone" command), you could of course simply create a new realdebrid remote and skip the additional virtualization layer of docker.

**Reply by mash2k3** [reply]
noted i will remove that option.

the binary version of rclone does not separate the media types by movies and shows, for kodi i need them separate.
Is it normal for so of the folders in mount to end with .mkv?

**Reply by itsToggle** [reply]
Are you using the official rclone binary version? You need to use my fork.
Some of the releases do have the file format in their name, yes
